README.md

```diff
@@ -257,7 +257,7 @@ Shutdown initiated by touching predefined file.
 
 ```php
 $restartFile = '/tmp/hermes_restart';
-$restart = Tomaj\Hermes\Restart\SharedFileRestart($restartFile);
+$restart = Tomaj\Hermes\Shutdown\SharedFileRestart($restartFile);
```

Comment:
Missed "restarts" in README
- `$restartFile`
- `$redisRestartKey`
- `SharedFileRestart`
- `RedisRestart`

Reply:
Thank you - fixed